Service/Runner.php

```diff
@@ -76,7 +76,7 @@ class Runner
     /**
      * @var array
      */
-    private $features = [];
+    private $tokenInfo = [];
```

Comment:
Tohle se mi moc nezdá, z jednoduchýho `string[]` se stává `mixed` s nedefinovanou strukturou. V testech na featury se používá `setFeatures(['featureA'])`, teď se bude muset konstruovat nějakej podivnej strukturovanej objekt. Ten API call je rychlej, navíc si nejsem jistej, jestli verifikace platnosti tam prostě nemá zůstat - pokud chceme střeva docker runneru vyjmout ven jako knihovnu, tak přesně sem ze to bude zase přidávat, protože `ActionController` i `Executor` nebudou součástí knihovny.

Reply:
Pokud bys to chtěl takhle optimalizovat, řešil bych to jako to bylo dřív v Transformacích nebo Orchestrátoru - cachovacím wrapperem okolo Storage API klienta.